packages/app-next-example-plugin/api-report.md

```diff
@@ -0,0 +1,20 @@
+## API Report File for "app-next-example-plugin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackstagePlugin } from '@backstage/frontend-plugin-api';
+import { default as React_2 } from 'react';
+
+// Warning: (ae-missing-release-tag) "examplePlugin" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
wouldn't mind slapping a `@public` on these just because, some time down the line :)